make_windows_package.sh

```diff
@@ -47,6 +47,7 @@ cp -R octsympy ${WINDIRTMP}
   ./bootstrap
   ./configure
   make )
+cp bin/winwrapy.bat ${WINDIRTMP}/bin/
```

Comment:
> > +cp bin/winwrapy.bat ${WINDIRTMP}/bin/
>
> what is this line for?
>
> —

Well, whenever i ran 'make_windows_package.sh' it always downloaded a fresh
copy of octsympy from git and packaged it. It did not include my
winwrapy.bat but the one from github. So, in order to include my edited
winwrapy.bat, i used this.